automation/build-artifacts.repos.el7

```diff
@@ -1,2 +1,3 @@
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs-modules_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
+http://jenkins.ovirt.org/job/ovirt-engine-yarn_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
```

Comment:
We have an open [ticket](https://ovirt-jira.atlassian.net/browse/OVIRT-886)  on fixing the issue with missing yum pkg.